src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
+
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/movie';
+
+type Props = {
+  movie: Movie[];
+};
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<Props> = ({ movie }) => (
+  <>
+    <div className="movies">
+      {movie.map(item => (
+        <MovieCard {...item} />
```

Comment:
do not use destructing here, it is not good practice, use it inside the MovieCard component